src/pages/careers/besquare/components/helper/_card.js

```diff
@@ -50,7 +50,7 @@ const Card = ({ card_content, custom_icon, has_list, style, title_component }) =
     return (
         <CardWrapper {...card_wrapper}>
             <IconWrapper {...icon_wrapper} src={custom_icon?.src || card_content.src} alt="" />
-            {title_component && title_component}
```

Comment:
Hi @niloo-fs, Any reason, you removed the availability check of this title_component ?

Reply:
All right! Looks like the check was unnecessary.